scalalib/src/giter8/Giter8Module.scala

```diff
@@ -0,0 +1,31 @@
+package mill.scalalib.giter8
+
+import mill.T
+import mill.define.{Command, Discover, ExternalModule}
+import mill.modules.Jvm
+import mill.scalalib.api.ZincWorkerUtil
+import mill.scalalib._
+import mill.BuildInfo
+import mill.api.Loose
+
+object Giter8Module extends ExternalModule with CoursierModule {
+
+  def init(args: String*): Command[Unit] = T.command {
```

Comment:
It would be nice, if we could return the final destination directory. But I'm not sure how easy it is to detect the correct location when the user is passing arbitrary options, so just take it as a suggestion, not a demand.

```suggestion
  def init(args: String*): Command[PathRef] = T.command {
```

Reply:
Not sure if parsing the output of command is sufficient? It outputs something like: `Template applied in C:\Users\sake_\AppData\Local\Temp\mill_giter8_test4305961942039948174` in one line.

Reply:
Hm, I think it's probably not worth it. Too easy to break in a next release. If we really need this output, I'd rather try to run g8 as library in-jvm and try to get the final path for it's API.

Reply:
Ok, I will make try to invoke its internal API, seems simple enough. 👍🏻

Reply:
I'm even not sure such an API exists, and the change to run in-JVM isn't trivial either. I don't stop you, but I think it's not worth it until there is need for it.

Reply:
Ok, so not that simple.. 😄  
Seems like they have these custom processors that we could hook in: https://github.com/foundweekends/giter8/blob/develop/app/src/main/scala/giter8.scala#L48-L58  
But this needs to be in a separate JAR for us to download it dynamically

Reply:
Yeah, agreed. It's not like users will be confused or something. 99.99% of the time they will `init` without any args, in current folder.  
And it's not like we need to cache the outputs or use it in another task..